spec.bs

```diff
@@ -0,0 +1,393 @@
+<pre class='metadata'>
+Title: Private Aggregation API
+Shortname: private-aggregation-api
+Level: 1
+Status: w3c/CG-DRAFT
+Group: patcg
+Repository: patcg-individual-drafts/private-aggregation-api
+URL: https://patcg-individual-drafts.github.io/private-aggregation-api
+Editor: Alexander Turner, Google https://www.google.com, alexmt@chromium.org
```

Comment:
```suggestion
Editor: Alexander Turner, Google Inc. https://www.google.com, alexmt@chromium.org
```

Reply:
The formal name actually changed to Google LLC in 2017, but I think just Google is probably fine (e.g. see https://wicg.github.io/trust-token-api/)

Reply:
I see, it's fine then. :)